lib/fable/events.ex

```diff
@@ -95,6 +194,7 @@ defmodule Fable.Events do
     end)
   end
 
+  @doc false
   def log(config, agg) do
```

Comment:
We do need docs for these functions somewhere.

Reply:
Yeah, I would've started by adding those as well to the module docs, maybe document the functions within `__using__`. I'm not really sure how useful it would eb to call functions directly on `Fable.Events`.

Reply:
What we could also do is add those functions to the behaviour. That's how ecto documents those functions and I guess they also don't really expect people to actually reimplement it (besides maybe for tests).